js/views/userPageVw.js

```diff
@@ -256,7 +256,35 @@ module.exports = baseVw.extend({
       if (e.guid === this.model.get('page').profile.guid) {
         this.renderUserUnblocked();
       }
-    });        
+    });  
+    
+    this.listenTo(window.obEventBus, 'saveCurrentForm', function(response){      
+      switch (response) {
+        case 'item':
+          self.saveItem();
+          break;
+        case 'customize':
+          self.saveCustomizePage();
+          break;
+      }
+    });   
+    
+    $(window).off('keydown.view').bind('keydown.view', function(e) {
+      var char = String.fromCharCode(e.which).toLowerCase(),
+          ctrl = (e.ctrlKey || e.metaKey) && !e.altKey;
+
+      if (ctrl && char == config.keyShortcuts.save) {
+        var state = '';
+        
+        if (self.itemEdit) {
+          state = 'item';
+        } else if (self.customizing) {
+          state = 'customize';
+        }
+        
+        window.obEventBus.trigger('saveCurrentForm', state);
```

Comment:
same as above... why trigger a global event, which AFAIK is being handled only within this same view?